app/configurator/map/map-chart-options.tsx

```diff
@@ -148,6 +150,49 @@ export const AreaLayerSettings = memo(
               disabled={!chartConfig.fields.areaLayer.show}
             ></ChartOptionSelectField>
           </ControlSectionContent>
+          <ControlSectionContent side="right">
+            <FieldSetLegend legendTitle={"Continuous"}></FieldSetLegend>
+            <ChartOptionRadioField
+              label="Linear interpolation"
+              field={activeField}
+              path="paletteType"
+              value={"continuous"}
+              disabled={!chartConfig.fields.areaLayer.show}
+            ></ChartOptionRadioField>
+            <FieldSetLegend legendTitle={"Discrete"}></FieldSetLegend>
+            <ChartOptionRadioField
+              label="Quantize (equal intervals)"
+              field={activeField}
+              path="paletteType"
+              value={"discrete"}
+              disabled={!chartConfig.fields.areaLayer.show}
+            ></ChartOptionRadioField>
+            <ChartOptionRadioField
+              label="Quantiles (equal distribution of values)"
+              field={activeField}
+              path="paletteType"
+              value={"quantile"}
+              disabled={!chartConfig.fields.areaLayer.show}
+            ></ChartOptionRadioField>
+            <ChartOptionRadioField
+              label="Jenks (natural breaks)"
+              field={activeField}
+              path="paletteType"
+              value={"jenks"}
+              disabled={!chartConfig.fields.areaLayer.show}
+            ></ChartOptionRadioField>
+            <ChartOptionSelectField
+              id="areaLayer.nbClass"
+              label="Number of classes"
+              field={activeField}
+              path="nbClass"
+              options={Array.from({ length: 7 }, (_, i) => 3 + i).map((d) => ({
```

Comment:
nit: Move the creation of this (constant) options array into a const outside of the component function. E.g.

```ts
const CLASSES_OPTIONS = Array.from(/* ... */)

export const AreaLayerSettings = () => {
/*...*/
<ChartOptionSelectField
  id="areaLayer.nbClass"
  options={CLASSES_OPTIONS}
/>
}
```